addons/project/data/mail_template_demo.xml

```diff
@@ -15,7 +15,7 @@
         <t t-out="user.signature or ''">--<br/>Mitchell Admin</t>
     </t>
 </div>
-<br/><span style="margin: 0px 0px 0px 0px; font-size: 12px; opacity: 0.5; color: #454748;">You are receiving this email because your project has been moved to the stage <b t-out="object.stage_id.name or ''">Done</b></span>
+<br/><span style="margin: 0px 0px 0px 0px; font-size: 12px; opacity: 0.5; color: #454748;" groups="project.group_project_stages">You are receiving this email because your project has been moved to the stage <b t-out="object.stage_id.name or ''">Done</b></span>
```

Comment:
@xavierbol Yes it works. We have added `project.group_project_stages` because it was giving the access error while previewing the template in mail composer wizard when "group_project_stages" is disabled in configuration.

Thanks :slightly_smiling_face:

Reply:
Okay great, thanks. 🙂